### DISCUSSION TITLE
Negative tags selection

### DISCUSSION THREAD

**Discussion by quazardous** [zim-desktop-wiki/zim-desktop-wiki#1997] [Ideas]
I love the way I can clic on tags and select combination to filter the pages. But IMO a negative filter could be usefull.

ie : using a tag `done`and a tag `task`

Display the todo list could be `task`and `! done`(not done)

**Comment by introt** [comment]
Nice idea!

This is already possible using the search dialog[^1], but exposing it via the tag plugin's side pane is way more accessible. For the specific use case in your example, there's also the task list plugin.

UX wise, I'd go for right-clicking, graying-out the text, and formatting it with strike-through && italics (something like ~_`this`_~).

[^1]: see [Keyword searches](https://zim-wiki.org/manual/Help/Searching.html); eg. `Tag: task AND NOT Tag: done` for the example given.

**Reply by quazardous** [reply]
I'm glad you like it !

For UX I'm more a green check/red cross guy but it's personal.
